src/content/docs/style-guide/images/embed-videos.mdx

```diff
@@ -34,12 +34,11 @@ Besides the docs site, Wistia hosts videos for [learn.newrelic.com](https://lear
 
 To use Wistia to host videos:
 
-1. Log on to [Wistia.com](https://wistia.com).
-2. Click **Projects** in the left navigation pane.
-3. In the center pane, page through until you find **docs.newrelic.com Videos**.
-4. Click **Add > Upload**.
-5. Click **Embed &#38; Share**.
-6. Copy the video ID from the embed code so you can insert it into your video link. For example, `jsh48c18dq` from this tag:
+1. Go to [Wistia.com](https://app.wistia.com/login/sso/new), type `newrelic`, and click **Continue to SSO**.
+2. Click **Content library > Content** and click **docs.newrelic.com Videos**.
```

Comment:
@ally-sassman I think we might still need to tell users to click through the pages until they find **docs.newrelic.com Videos**--unless there's a faster way to find it.

Reply:
@rhetoric101 I don't image users see a ton of options. I only see this one project. Are you thinking users see more projects if they're involved with other teams?